cfgov/v1/jinja2/v1/landing-page-2/_template-prototype-text-intro-content.html

```diff
@@ -0,0 +1,7 @@
+<h1 class='m-text-introduction_heading'>Text Introduction</h1>
```

Comment:
If this content is always going to be consistent, should we be using individual templates? For the hero and half-width blobs, I passed in the chunks as a setting argument to restrict what can be put in, and to limit our use of individual templates, allowing the CMS to do all the work. Ex. https://github.com/cfpb/cfgov-refresh/pull/1064/files#diff-fa9a252fc9441500838f4cb4baad3cdfR11